### DISCUSSION TITLE
I am trying to deploy a sample web-classification flow to azure app service but instructions are either incomplete or incorrect.

### DISCUSSION THREAD

**Discussion by niveditau** [microsoft/promptflow#1190] [Q&A]
Once you have created a connection with azure open ai and generated a 'dist' folder and trying to build a docker image it gets stuck at 
`COPY ./connections/* /connections/` because there was no connections directory generated inside dist directory. I tried creating it manually and it moves forward with the docker image creation but then fails to create a connection at run time. Is here anyone able to successfully deploy the sample using docker even if just locally? Please share your work arounds. Thanks!

**Comment by wangchao1230** [comment]
Thanks for reaching out to us. 
I reproed the issue and we will take a look.

**Comment by iljoong** [comment]
Since the documentation is missing connection setup, I had the same problem. Here is the workaround.

First move to `/runit/promptflow-serve` folder in the `dist` folder and open the `run` file.
You'll find how promptflow ininitalizes the app and you'll also see one of more `pf connection create --file ...` lines, just like below.

```
pf connection create --file /connections/azure_openai_connection.yaml
```

The name of connection file is the __name of the connection__ used in the promptflow so it may differ in your file.

You also need copy the connection template file and update the value of api_key/api_base to environment variable placeholder.

For `azure_openai_connection.yaml` file example,

```
$schema: https://azuremlschemas.azureedge.net/promptflow/latest/AzureOpenAIConnection.schema.json
name: azure_open_ai_connection
type: azure_open_ai
api_key: "${env:AZURE_OPENAI_KEY}"
api_base: "${env:AZURE_OPENAI_BASE}"
api_type: "azure"
```

In case you have more than one connections, you need to do the same step.

Last, you need to set the right value of environment variable in the Web App Configuration or provide environment variable parameters for local docker run.

Hope this helpful!

**Comment by YingChen1996** [comment]
Hi, @niveditau. The fix is merged https://github.com/microsoft/promptflow/pull/1242 and will be released in the next stable version.
